src/server/chromium/crBrowser.ts

```diff
@@ -361,7 +361,7 @@ export class CRBrowserContext extends BrowserContext {
   async _doCookies(urls: string[]): Promise<types.NetworkCookie[]> {
     const { cookies } = await this._browser._session.send('Storage.getCookies', { browserContextId: this._browserContextId });
     return network.filterCookies(cookies.map(c => {
-      const copy: any = { sameSite: 'None', ...c };
+      const copy: any = { sameSite: 'Lax', ...c };
```

Comment:
I don't understand this line at all. Do we lack sameSite property sometimes? Why do we assume Lax default now?

Reply:
lax is default now in chromium unless specified explicitly. (Used to be 'None')